### ISSUE TITLE
Several icons not listed on Maki website

### ISSUE THREAD

**Issue by danopia** [mapbox/maki#592]
Hello, I've noticed that there are several icons in this git repo that are not represented on [the Maki website](https://labs.mapbox.com/maki-icons/) and editor. I see **204 icons** on the website and **213 icons** in git.

I compared the list of icons from both places and identified these icons that are missing from the website:

* `animal-shelter.svg`
* `gate.svg`
* `historic.svg`
* `lift-gate.svg`
* `nightclub.svg`
* `parking-paid.svg`
* `terminal.svg`
* `toll.svg`
* `tunnel.svg`

Some of these icons appear at least a few years old. I imagine this isn't intentional.. just bringing this up in case it can be rectified.

**Comment by rotsee** [comment]
The web page seems like it hasn't been maintained for ages. The npm installation instructions are increadibly outdated, for instance (will install a version of the icon-set from 2016 !)

It would be great to get an hint from Mapbox on wether the whole Maki icons support is now abandonware, or something that will keep getting developed.

**Comment by danopia** [comment]
Good point. I just checked the timestamp on the website's JS file and it says `Mon, 12 Feb 2024 22:36:54 GMT` which implies that the site was rebuilt or at least touched in the past year. So there might be hope

**Comment by theadventurer62** [comment]
> Good point. I just checked the timestamp on the website's JS file and it says `Mon, 12 Feb 2024 22:36:54 GMT` which implies that the site was rebuilt or at least touched in the past year. So there might be hope

According to a reply in another issue, this was done only because the page was down
